develop/items/first-item.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Creating Your First Item
+description: Learn how to register a simple item and how to texture, model and name it.
+authors:
+  - IMB11
+---
+
+# Creating Your First Item
+
+This page will introduce you into some key concepts relating to items, and how you can register, texture, model and name them.
+
+If you aren't aware, everything in Minecraft is stored in registries, and items are no exception to that.
+
+## Preparing Your Items Class
+
+To simplify the registering of items, you can create a method that accepts an instance of an item and a string identifier.
```

Comment:
Maybe explain what this method does? As I said bellow we should be consitent with how we register stuff.